src/components/UI/Progress/CircularProgress.vue

```diff
@@ -43,35 +47,49 @@
 <script>
 export default {
   name: "CircularProgress",
+  data() {
+    return {
+      progressBarBackgroundColor: "#C4C5C5",
+      progressBarForegroundColor: "#2B9D05",
+    };
+  },
   props: {
     radius: Number, // radius of ring
-    progress: Number, // progress of the ring
+    progressBarPercent: Number, // progress of the ring
     stroke: Number, // thickness of the ring
-    progressNumberIndicator: {
-      // to show a number indicator in between and if yes
-      // what additional text to show along with it
+    /**to show a result in the center of the progress bar
+     * and if enabled, what is the title of the result to show
+     * along with it
+     */
+    result: {
       default: () => {
         return {
           enabled: false,
-          additionalText: "",
+          title: "",
         };
       },
       type: Object,
     },
   },
   computed: {
+    /** Calculating a reduced radius as we don't want the ring to overflow
+     * the svg viewbox if the stroke is high
+     * Reference - https://css-tricks.com/building-progress-ring-quickly/
+     */
     normalizedRadius() {
-      // refer - https://css-tricks.com/building-progress-ring-quickly/
       return this.radius - this.stroke * 2;
     },
+    // Circumference of the inner circle of the ring
     circumference() {
```

Comment:
can the name be more specific - like `innerRingCircumference`?

Reply:
sorry this is not the inner righ circumference, I've corrected the comment